operator/main.go

```diff
@@ -256,5 +259,7 @@ func defineFlagVar() *FlagVar {
 		"Timeout of Read / Write for the pprof server.")
 	flag.DurationVar(&flagVar.cacheSyncTimeout, "cache-sync-timeout", defaultCacheSyncTimeout,
 		"Indicates the cache sync timeout in seconds")
+	flag.StringVar(&flagVar.installTarget, "install-target", string(prepare.InstallTypeSecret),
```

Comment:
this should not be passed as an argument here but be part of the library / setup call for module manager imo. Also I think it can be hardcoded since you currently only have 1 valid strategy, the other meant for testing

Reply:
We can think about it, but currently there are two ways - change the library interface (which is publicly available and dangerous), configuring the operator, which is for our internal consumption, I chose the later.

Reply:
I suggest changing the library if you already code the lookup logic. One common way that this is solved in libraries is to implement a lookup function as a passed parameter.

Reply:
this implementation was changed as discussed.